api/egl.xml

```diff
@@ -1110,6 +1110,11 @@
         <enum value="0x8F74" name="EGL_CLIENT_PIXMAP_POINTER_HI"/>
     </enums>
 
+    <enums namespace="EGL" start="0x3570" end="0x3580" vendor="Google" comment="Reserved for Tom Murphy (Khronos bug ...)">
```

Comment:
This block should move back to before the "<!-- Please remember" comment ca. line 1092.